bzt/modules/gatling.py

```diff
@@ -612,6 +630,8 @@ def _extract_log_gatling_22(self, fields):
         if fields[0].strip() == "GROUP":
             return self.__parse_group(fields)
         elif fields[0].strip() == "REQUEST":
+            if self.guessed_gatling_version == "3.X":
+                fields.insert(1, 'Taurus Scenario')
```

Comment:
Is it always "Taurus Scenario"? I guess it can be different for user's scala files